pkg/liqonet/iptables/iptables.go

```diff
@@ -570,6 +641,68 @@ func (h IPTHandler) insertRulesIfNotPresent(table, chain string, rules []string)
 	return nil
 }
 
+func getPostroutingRules(isGateway bool, tep *netv1alpha1.TunnelEndpoint) ([]string, error) {
+	clusterID := tep.Spec.ClusterID
+	if clusterID == "" {
+		return nil, &errors.WrongParameter{
+			Parameter: consts.ClusterIDLabelName,
+			Reason:    errors.StringNotEmpty,
+		}
+	}
+	localPodCIDR := tep.Status.LocalPodCIDR
+	localRemappedPodCIDR, remotePodCIDR := utils.GetPodCIDRS(tep)
+	if err := utils.IsValidCIDR(localPodCIDR); err != nil {
```

Comment:
You have introduced the validations of the parameters before trying to insert the rules, that's a nice! What do you think if we introduce a function to check the correctness of the `tep`? Such way we do it once.